go.mod

```diff
@@ -1,38 +1,119 @@
 module github.com/cloudflare/sciuro
 
-go 1.16
+go 1.17
```

Comment:
Mostly out of curiosity, is Bazel compatible with go1.18 ?

Reply:
rules_go 0.32.0 is compatible with Go 1.18 (but that's not really what this command in go.mod effects)